src/model/index.ts

```diff
@@ -85,11 +90,17 @@ export function models() {
     actions: DummyColumn,
     annotate: AnnotateColumn,
     selection: SelectionColumn,
-
+    heatmapcustom: MultiValueColumn,
+    sparkline: MultiValueColumn,
     max: MaxColumn,
     min: MinColumn,
     mean: MeanColumn,
     script: ScriptColumn,
-    nested: NestedColumn
+    nested: NestedColumn,
+    threshold: MultiValueColumn,
```

Comment:
why do you still need multiple model classes for multi values. With the suggested changes to introduce an optional field `rendererType` you can set the default renderer within the description but the model class is the same